layouts/tags/list.html

```diff
@@ -0,0 +1,77 @@
+{{ define "title" }}
+<div class="page-banner standard">
+  <div class="container">
+    <div class="page-banner--title">
+      <h1>{{ i18n "tagged-with" }}: {{ .Title }}</h1>
+    </div>
+  </div>
+</div>
+{{ end }}
+
+{{ define "main" }}
+<div class="container">
+  <div class="row">
+    <section class="col-md-8">
+      <ul id ="blog-list-result" class="blogpost-list">
+        {{ $currentTag := .Title}}
+        {{ range .Pages }}
+        {{ if in .Params.tags $currentTag }}
+        <li class="post">
+          <article class="post-container">
+            <div class="text-container">
+              <div style="margin-top: 1rem;">
+                <a href="{{ .Permalink }}" class="blog-title">
+                  <h2 style="font-size: 2.2rem; font-weight:600;">{{ .Title | markdownify }}</h2>
+                </a>
+              </div>
+              <div class="date">
+                <time datetime='{{ .Date.Format "2006-01-02T15:04:05Z07:00" }}'>{{ .Date.Format "January 2, 2006" }}</time>
```

Comment:
Should these two hardcoded dates be read from the post object?

Reply:
The code reads the date directly from the post object. Specifically, it accesses the Date property of the post object within the range loop. The expression .Date.Format formats the date in the specific pattern in quotes.